### ISSUE TITLE
Incorrect resolve of spritesheet image located at domain root

### ISSUE THREAD

**Issue by rasentry** [pixijs/pixi.js#3715]
spritesheet located in domain root directory for example
http://test.com/spritesheet.json
with
http://test.com/spritesheet.png
when i pass '/spritesheet.json' to pixi loader it takes base dir, and concatenates with '/' and 'spritesheet.png'. As a result I have '//spritesheet.png' which treated by browser as http://spritesheet.png

It happens here https://github.com/pixijs/pixi.js/blob/dev/src/loaders/spritesheetParser.js#L38

**Comment by bigtimebuddy** [comment]
Good catch. 

Looks like the spritesheet parser doesn't support root-relative paths. We could probably tweak that little bit to do something like this: 

```js
const basePath = path.dirname(resource.url.replace(this.baseUrl, ''));
// handle root-relative paths
if (basePath === '/') {
    basePath = '';
}
resourcePath = `${basePath}/${resource.data.meta.image}`;
```

Not sure if this is the best approach, but see if tweaking that works for you.

**Comment by bigtimebuddy** [comment]
Merged #3723
